firebase-perf/src/main/java/com/google/firebase/perf/logging/ConsoleUrlGenerator.java

```diff
@@ -0,0 +1,58 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.perf.logging;
+
+public final class ConsoleUrlGenerator {
+  private static final String URL_BASE_PATH = "https://console.firebase.google.com";
+  private static final String UTM_MEDIUM = "perf-android-studio";
```

Comment:
Let not hardcoded it to android-studio. Developer do developer Android applications on other IDEs.

How about calling it "perf-android-ide"?

Reply:
Good catch! Done. I have also added `utm_source` because I have tested that GA needs both `source` and `medium` for logging visits. PTAL.